float64 forceTarget
float64 forceNow
float64 paraB
float64 deltaB
float64 zdd
float64 commandVel
float64 actualVel